### ISSUE TITLE
Split tests from libOptions.js

### ISSUE THREAD

**Issue by coderaiser** [coderaiser/minify#76]
Split tests from [`test/libOptions.js`](https://github.com/coderaiser/minify/blob/master/test/libOptions.js) so there was one assertion and `t.end()` similar to [`test/minify`](https://github.com/coderaiser/minify/blob/master/test/minify.js)

**Comment by code-forger** [comment]
Ahh I forgot to call t.end().. embarrassing :D

I'm not sure I see the value in splitting these tests such that each only has one assertion though?

The assertions in place ensure that the single call made by each unit went exactly as expected.?

**Comment by coderaiser** [comment]
Using `test.end()` it’s a convention started from `tape`, yes we can avoid it, and there is such place in [@putout/test](https://github.com/coderaiser/putout/tree/v20.4.0/packages/test#processors-api). But it uses destructructoring on test function argument, because:
- one assertion per test, [**Single Responsibility Principle**](https://en.wikipedia.org/wiki/Single-responsibility_principle): *Do one thing and do it well do ☝️*;
- when you have only one assertion, what purpose of using variable `t`? It just have no sense;
- 

```js
test(‘minify: options: no’, ({deepEqual}) => {
    const error = assign(Error(‘Cannot read file’), {
        code: ‘ENOENT’
    });
    const options = readOptions({
        readFile: stub().throws(error), 
    });
   
   const expected = {};
   
    deepEqual(data, expected);
});
```

But I don’t know either it’s more readable 🤷🏽  then:

```js
test(‘minify: options: no’, ({deepEqual}) => {
    const error = assign(Error(‘Cannot read file’), {
        code: ‘ENOENT’
    });
    const options = readOptions({
        readFile: stub().throws(error), 
    });
   
   const expected = {};
   
    t.deepEqual(data, expected);
    t.end(); 
});
```

`t.end()` shows that there will be no more assertions, only one is allowed.

**Comment by coderaiser** [comment]
Landed in [v7.2.0](https://github.com/coderaiser/minify/releases/tag/v7.2.0) 🎉
